5_People/nitean-recognition.md

```diff
@@ -2,23 +2,47 @@
 
 As Niteans, we believe in celebrating small wins and recognizing achievements. By enabling Nitean recognition, we're boosting each other's morale, enhancing engagement and communication, and improving the overall company culture.
 
-## Wall of Happy
+## Peer recognition
 
-This is inspired by head of Google’s People Operations, [Laszlo Bock](https://plus.google.com/+LaszloBock/posts/UzxkRpkvyf7). He has a Wall of Happy outside his office where kudos are printed and put on a physical wall.
+To enable peer recognition, we're using the [Bonus.ly](https://bonus.ly/) platform which enables Niteans to give each other micro-bonuses in the form of +1s. Big or small of an act, if it made you happy, recognize and reward it.
 
-Instead of a physical wall, we have a Slack channel `wall-of-happy` where you can see a steady stream of kudos and reactions, enabled by the [Bonus.ly](https://bonus.ly/) peer recognition platform. Monitor leader board here: https://bonus.ly/analytics/leaderboards.
+We took a page from the head of Google’s People Operations, Laszlo Bock, and created our own version of the [_wall-of-happy_](https://plus.google.com/+LaszloBock/posts/UzxkRpkvyf7). Instead of a physical wall where kudos are printed out, we have a Slack channel `recognition` where you can see a steady stream of kudos and reactions.
+
+Monitor the leader board here: https://bonus.ly/analytics/leaderboards.
 
 ## Policy
 
-Niteans get 10 points allowance every month. We're adopting the `+1` system. This means that whenever you give someone a bonus, you give that person 1 point.
+Niteans get 30 points allowance every month. We're adopting the `+1` system. This means that whenever you give someone a bonus, you give that person 1 point. The equivalent value of a point is 10EUR.
```

Comment:
I would remove `The equivalent value of a point is 10EUR.` as this is not true. It's not an equivalent value, it's merely a helping thought exercise, in some cases, should you need one.